### PR TITLE
feat: create 2021-11-12-Community-call-gestire-soluzione-riuso

### PR DESCRIPTION
**Pull request by Francescopog**
## Description

This PR tackles:

* ...
* ...
* ...

In particular, the ...

## Checklist

* [ ] I followed the indications in [CONTRIBUTING.md](https://github.com/italia/developers.italia.it/blob/master/CONTRIBUTING.md)
* [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?
* [ ] Ready for review! :rocket:

## Fixes

Fixes #

### CODE REVIEW

File: _posts/it/2021-11-12-Community-call-gestire-soluzione-riuso.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ 
+Durante l’incontro, Marco Spasiano (CNR), Cristian Lucchesi (CNR) e Ciro Salvio (Istituto Superiore di Sanità) ci illustreranno i casi di cessione e riuso di ePAS e SIGLA raccontando la loro storia, in qualità di **autore e fruitori** dell’open source italiano. Una storia caratterizzata da comuni esigenze: le necessità iniziali, la scelta della soluzione da adottare, la sua configurazione e la sua messa in produzione.
+
+La community call sarà incentrata in particolare sulle **modalità di gestione delle soluzioni in riuso**: la conduzione dell’analisi comparativa, la scelta a catalogo delle soluzioni e la costruzione del rapporto con l’ente cedente. Infine verranno messi in luce i vantaggi di fare rete e, grazie alla partecipazione attiva, l’opportunità di costruire una comunità di confronto.
+
+In chiusura, i partecipanti avranno spazio per soddisfare tutte le curiosità e dubbi in un momento Q&A.
+
+**[Registrati](https://mobilizon.it/events/fdffc27c-2046-4719-bbf4-11282669cd6f){:target="_blank"} all’evento per partecipare**. Ti aspettiamo il 24 novembre!
```
Comment (bfabio) [inline _posts/it/2021-11-12-Community-call-gestire-soluzione-riuso.md:24]:
```suggestion
**[Registrati all'evento](https://mobilizon.it/events/fdffc27c-2046-4719-bbf4-11282669cd6f){:target="_blank"} per partecipare**. Ti aspettiamo il 24 novembre!**
```

File: _posts/it/2021-11-12-Community-call-gestire-soluzione-riuso.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: " Community call: come si gestisce una soluzione a riuso"
```
Comment (bfabio) [inline _posts/it/2021-11-12-Community-call-gestire-soluzione-riuso.md:3]:
```suggestion
title: "Community call: come si gestisce una soluzione a riuso"
```

**Review by bfabio — CHANGES_REQUESTED** [review]

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  There is a problem with the commit message
> Update _posts/it/2021-11-12-Community-call-gestire-soluzione-riuso.md

Co-authored-by: Fabio Bonelli <fb@fabiobonelli.it>
- subject may not be empty
- type may not be empty
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  There is a problem with the commit message
> Update _posts/it/2021-11-12-Community-call-gestire-soluzione-riuso.md

Co-authored-by: Fabio Bonelli <fb@fabiobonelli.it>
- subject may not be empty
- type may not be empty
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 03a74d777780264349ea8ea4f84e085fab6cc05e
</p>

File: _posts/it/2021-11-12-Community-call-gestire-soluzione-riuso.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+ 
+Durante l’incontro, Marco Spasiano (CNR), Cristian Lucchesi (CNR) e Ciro Salvio (Istituto Superiore di Sanità) ci illustreranno i casi di cessione e riuso di ePAS e SIGLA raccontando la loro storia, in qualità di **autore e fruitori** dell’open source italiano. Una storia caratterizzata da comuni esigenze: le necessità iniziali, la scelta della soluzione da adottare, la sua configurazione e la sua messa in produzione.
+
+La community call sarà incentrata in particolare sulle **modalità di gestione delle soluzioni in riuso**: la conduzione dell’analisi comparativa, la scelta a catalogo delle soluzioni e la costruzione del rapporto con l’ente cedente. Infine verranno messi in luce i vantaggi di fare rete e, grazie alla partecipazione attiva, l’opportunità di costruire una comunità di confronto.
+
+In chiusura, i partecipanti avranno spazio per soddisfare tutte le curiosità e dubbi in un momento Q&A.
+
+**[Registrati all'evento](https://mobilizon.it/events/fdffc27c-2046-4719-bbf4-11282669cd6f){:target="_blank"} per partecipare**. Ti aspettiamo il 24 novembre!**
```
Comment (bfabio) [inline _posts/it/2021-11-12-Community-call-gestire-soluzione-riuso.md:24]:
```suggestion
**[Registrati all'evento](https://mobilizon.it/events/fdffc27c-2046-4719-bbf4-11282669cd6f){:target="_blank"} per partecipare. Ti aspettiamo il 24 novembre!**
```

**Review by bfabio — APPROVED** [review]

**Pull request merged**